README.md

```diff
@@ -183,6 +192,28 @@ Note that `+Private` files can only be used with ObjC++ source (other headers ar
 Enums are translated to C++ `enum class`es with underlying type `int`, ObjC `NS_ENUM`s with
 underlying type `NSInteger`, and Java enums.
 
+### Flags
+Flags are translated to C++ `enum class`es with underlying type `unsigned` and a generated set
+of overloaded bitwise operators for convenience, ObjC `NS_OPTIONS` with underlying type
+`NSUInteger`, and Java enums. Contrary to the above enums, the enumerants of flags represent
+single bits instead of integral values.
+
+When specifying a `flags` type in your IDL file you can assign special semantics to options:
+
+```
+my_flags = flags {
+  flag1;
+  flag2;
+  flag3;
+  no_flags = none;
+  all_flags = all;
+}
+```
+In the above example the elements `no_flags` and `all_flags` are given special meaning.
```

Comment:
It's unclear to me here whether it's "no_flags" or "none" which has the special meaning here.  My best guess (not having read the code yet) is that "= all" is the sequence which applies the special meaning, while "no_flags" is just a name.  Is that correct?  If so, that could be clarified in the text.  Or you could replace "no_flags" with something which is more clearly an example of user-supplied value like "my_no_flags" or "none_of_my_flags".